packages/e2e-testing/chain-setup.ts

```diff
@@ -3,33 +3,40 @@ import {ETHERLIME_ACCOUNTS, GanacheServer} from '@statechannels/devtools';
 import {utils} from 'ethers';
 import {deploy as deployNitro} from '@statechannels/server-wallet/lib/deployment/deploy';
 import {deploy as deployGraph} from '@graphprotocol/statechannels-contracts/deployment/deploy';
-import _ from 'lodash';
 
 export default async function setup(): Promise<void> {
+  if (process.env.CHAIN_NETWORK_ID) {
+    console.log(
+      `CHAIN_NETWORK_ID defined as ${process.env.CHAIN_NETWORK_ID}. Assuming chain env vars are set by caller`
+    );
+    return;
+  }
+
+  process.env['CHAIN_NETWORK_ID'] = '9002';
   process.env['GANACHE_HOST'] = '0.0.0.0';
   process.env['GANACHE_PORT'] = '8545';
   process.env[
     'RPC_ENDPOINT'
   ] = `http://${process.env['GANACHE_HOST']}:${process.env['GANACHE_PORT']}`;
-
   const ethPerAccount = utils.parseEther('100').toString();
   const etherlimeAccounts = ETHERLIME_ACCOUNTS.map((account) => ({
     ...account,
     amount: ethPerAccount
   }));
-  const defaultServerWalletAccount = {
-    privateKey: process.env.ETHEREUM_PRIVATE_KEY // useful if using rinkeby account for example
-      ? process.env.ETHEREUM_PRIVATE_KEY
-      : ETHERLIME_ACCOUNTS[0].privateKey,
-    amount: ethPerAccount
-  };
-  const accounts = _.unionBy(etherlimeAccounts, [defaultServerWalletAccount], 'privateKey');
 
   if (!process.env.GANACHE_PORT) {
     throw new Error('process.env.GANACHE_PORT must be defined');
   }
-  const ganacheServer = new GanacheServer(parseInt(process.env.GANACHE_PORT), 1337, accounts);
+  const ganacheServer = new GanacheServer(
+    parseInt(process.env.GANACHE_PORT),
+    Number(process.env.CHAIN_NETWORK_ID),
+    etherlimeAccounts,
+    10_000, // timeout
+    10_00_000_000, // gasLimit
+    1 // gasPrice
```

Comment:
Why are these changed?

Reply:
Oops this was copied from the `chain-setup` in `server-wallet`. The important bit is that we don't use `1337` as the chain id.